vllm/entrypoints/llm.py

```diff
@@ -545,11 +545,13 @@ def _run_engine(
                 total=num_requests,
                 desc="Processed prompts",
                 dynamic_ncols=True,
-                postfix=f"Generation Speed: {0:.2f} toks/s",
+                postfix=(f"Prefill Speed: {0:.2f} toks/s, "
+                         f"Generation Speed: {0:.2f} toks/s"),
```

Comment:
Possibly we could remove "Speed" from both of these to take up less space?

Reply:
Yes!